sky/serve/replica_managers.py

```diff
@@ -600,6 +640,10 @@ def _download_and_stream_logs(info: ReplicaInfo):
         p.start()
         self._down_process_pool[replica_id] = p
 
+        if replica_id in self._launch_process_pool:
+            info.status_property.sky_launch_status = ProcessStatus.INTERRUPTED
+            del self._launch_process_pool[replica_id]
```

Comment:
Why separate this into two `if` branch? Can we combine them?

Reply:
what do you mean by _two_ `if` branch? There seems to be only one if?

Reply:
let me merge them